Common/src/geometry_structure.cpp

```diff
@@ -8090,10 +8176,14 @@ void CPhysicalGeometry::Read_SU2_Format_Parallel(CConfig *config, string val_mes
   
   map<unsigned long,bool> ElemIn;
   map<unsigned long, bool>::const_iterator MI;
+
 #ifdef HAVE_MPI
 #ifdef HAVE_PARMETIS
-  /*--- Initialize a vector for the adjacency information (ParMETIS). ---*/
-  vector< vector<unsigned long> > adj_nodes(nPoint, vector<unsigned long>(0));
+  /*--- Resize the vector for the adjacency information (ParMETIS). ---*/
+  vector< vector<unsigned long> >::iterator it2;
+  adj_nodes.resize(nPoint);
+  for (it2 = adj_nodes.begin(); it2 != adj_nodes.end(); it2++)
```

Comment:
Maybe it would be possible to do a adj_nodes.clear() before resizing and then use the second argument of resize since only new object in resize are initialized with the second argument.

Reply:
Ah that answers it, the vector may already contain something.
Yeah clear and resize or std::fill sound good.